sources/updater/waves/README.md

```diff
@@ -0,0 +1,39 @@
+# Update waves
+
+As mentioned in the [Bottlerocket update overview](../README.md), OS updates can include "waves" for staggered deployment.
+These waves are defined in the `manifest.json`, which lives in the TUF repository.
+Each time an OS update is made available, `manifest.json` is updated with the information pertinent to that update using [updata](../updog) or its related libraries.
+Waves may be supplied to `updata` on the command line, passed as a TOML-formatted file.
+
+This directory contains a few examples of these update wave files.
+Specific details are encapsulated in each file, but they are:
+
+* `default-waves.toml`: A "normal" deployment
+* `accelerated-waves.toml`: An accelerated deployment
+* `ohno.toml`: An extremely accelerated deployment in case of emergency.
+
+## Understanding the concept of waves
+
+Waves include a *seed* and a *start time*.
+
+Each Bottlerocket node generates a "seed" for itself which is simply a number between 0-2048 that determines where it falls in the update order.
+Nodes that have a seed within the current wave will update.
+All waves include the seeds of the prior wave, so if a node misses its wave for whatever reason, it still updates at a later time.
+
+The "start time" of a wave signals the end of the current wave and the beginning of the next wave.
+
```

Comment:
```suggestion
```
This makes it sound like the start time of a wave is the start time of the next wave, which is weird, and recursive without a base case :P